riscv-asm.md

```diff
@@ -545,7 +545,12 @@ mv rd, rs                    | addi rd, rs, 0
 not rd, rs                   | xori rd, rs, -1                                               | One’s complement
 neg rd, rs                   | sub rd, x0, rs                                                | Two’s complement
 negw rd, rs                  | subw rd, x0, rs                                               | Two’s complement word
+sext.b rd, rs                | slli rd, rs, XLEN - 8; srai rd, rd, XLEN - 8                  | Sign extend byte | It will expand another instruction sequence when B extension is available*[1]
```

Comment:
```suggestion
sext.b rd, rs                | slli rd, rs, XLEN - 8; srai rd, rd, XLEN - 8                  | Sign extend byte | It will expand to another instruction sequence when B extension is available*[1]
```